src/reference/asciidoc/whats-new.adoc

```diff
@@ -106,7 +106,7 @@ See "`<<jdbc>>`" for more information.
 === FTP and SFTP Changes
 
 A `RotatingServerAdvice` is now available to poll multiple servers and directories with the inbound channel adapters.
-See "<<ftp-rotating-server-advice>>" and "<<sftp-rotating-server-advice>>" for more information.
+See "<<ftp-rotating-server-advice>>" and <<sftp-rotating-server-advice>> for more information.
 
```

Comment:
Why quotes on one but not the other?

Reply:
No idea. There should be quotation marks on neither of them.